tdp/core/models/stale_component.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+from sqlalchemy import Column, String, Boolean
+
+from tdp.core.models.base import Base
+from tdp.core.operation import SERVICE_NAME_MAX_LENGTH, COMPONENT_NAME_MAX_LENGTH
+
+
+class StaleComponent(Base):
+    """Hold what component version are deployed.
```

Comment:
It is a copy/paste from `ComponentVersionLog` class ?
```suggestion
    """Hold what service component are staled.
```